features/cli/deploy.feature

```diff
@@ -46,7 +46,10 @@ Feature: deployment related features
     Then the output should contain "rolled out"
     # Wait the deployment till complete
     And the pod named "hooks-1-deploy" becomes ready
+    Given I wait up to 60 seconds for the steps to pass:
```

Comment:
@zhouying7780 you don't need to wrap it around with wait because the `I wait for` step already has a timeout

Reply:
@pruan-rht  Thanks , have updated.